iphone/cli/commands/_build.js

```diff
@@ -208,14 +208,15 @@ iOSBuilder.prototype.assertIssue = function assertIssue(issues, name) {
 };
 
 /**
- * Retrieves the certificate information by name.
+ * Retrieves the list of certificate information by name.
  *
  * @param {String} name - The cert name.
  * @param {String} [type] - The type of cert to scan (developer or distribution).
- * @returns {Object|null}
+ * @returns {Array}
  * @access private
  */
 iOSBuilder.prototype.findCertificate = function findCertificate(name, type) {
```

Comment:
Should be plural since it returns multiple certificates now.

Reply:
fixed.